src/main/java/de/retest/web/util/SeleniumWrapperUtil.java

```diff
@@ -41,21 +44,37 @@ public static boolean isWrapper( final WrapperOf w, final Object o ) {
 	}
 
 	/**
+	 * Extracts the wrapped element or driver from a given object while implementing backwards compatibility to the
+	 * internal selenium api. It is advised to call {@link #isWrapper(WrapperOf, Object)} before calling this method.
```

Comment:
Minor/optional (sorry, forgot previously):

```suggestion
	 * internal Selenium API. It is advised to call {@link #isWrapper(WrapperOf, Object)} before calling this method.
```